tests/v2/test_utils_keyfile.nim

```diff
@@ -0,0 +1,135 @@
+{.used.}
+
+import
```

Comment:
Does this file cover all the unit tests included in https://github.com/status-im/nim-eth/blob/master/tests/keyfile/test_keyfile.nim ?